setup.cfg

```diff
@@ -30,6 +30,7 @@ classifiers =
 install_requires =
     boto3
     boto3-stubs[apigateway,cognito,essential]
+    joblib
```

Comment:
In order to avoid the [dependency confusion vulnerability](https://medium.com/@alex.birsan/dependency-confusion-4a5d60fec610), we should have a version pin on `joblib`, preferably using the `~=` pessimistic operator.

(For that matter, there should be version pins on _all_ the package dependencies.)

Reply:
I've added the missing version pins based on the versions in my development environment. Let me know if it looks ok now.